decentraland/unity-explorer
# Feat: sdk7 audio source url change

- add support to clip url change for audioSource component
- now AudioSource is played only if player is inside the scene (muted otherwise)

# How to test
- start experience and verify there is no any music playing from the start
- go to Dance Floor and verify music is only played if when you inside the dance floor
